hat/assets/js/apps/Iaso/index.js

```diff
@@ -29,39 +29,6 @@ export default function iasoApp(
     userHomePage,
 ) {
     const plugins = getPlugins(enabledPluginsName);
-    const allRoutesConfigs = [
-        ...routeConfigs,
-        ...plugins.map(plugin => plugin.routes).flat(),
-    ];
-    const baseRoutes = allRoutesConfigs.map(routeConfig => (
-        <Route
-            path={getPath(routeConfig)}
-            component={
-                routeConfig.allowAnonymous
-                    ? routeConfig.component
-                    : props => (
-                        <ProtectedRoute
-                              {...props}
-                              featureFlag={routeConfig.featureFlag}
-                              permissions={routeConfig.permissions}
-                              component={routeConfig.component(props)}
-                              isRootUrl={routeConfig.isRootUrl}
-                              allRoutes={allRoutesConfigs}
-                          />
-                      )
-            }
-        />
-    ));
-
-    const overrideLandingRoutes = plugins
-        .filter(plugin => plugin.overrideLanding)
-        .map(plugin => plugin.overrideLanding);
-    // using the last plugin override (arbitrary choice)
-    const overrideLanding =
-        overrideLandingRoutes.length > 0
-            ? overrideLandingRoutes[overrideLandingRoutes.length - 1]
```

Comment:
Out of curiosity: what was gained by moving this code to `App`?

Reply:
so i could use current user reducer